### ISSUE TITLE
Unbalanced weapons for EVE factions?

### ISSUE THREAD

**Issue by eriktq1** [DeadAirRT/deadairfactionmindatar#3]
Hi, first of all i want to thank you for making some sort of dream come true.
Always wanted to play whit the EVE ships in the X4 galaxy.
Now to the thing:
I feel the weapons for the new factions are very powerful, almost too powerful? Or am i just imagining it? I mean specially the amarr ones seem to just chew thru anything like crazy. 
I run this whit VRO, XR shippack and your own scriptpackages (plus some QOL mods) if that is of interest.

And a couple of questions and comments:
Any plans for S and M ships?
The beam for amarr Standard and MultiF seems rather "wide".
The sound of Gallente blasters feels a little "off"? :)

Anyways, just thought id share some thoughts. Thank you again for the fantastic work!

**Comment by DeadAirRT** [comment]
Thanks for the feedback. I've been adjusting the balancing as I've been going but as you can imagine, it's hard to balance hitscan weapons to not be overpowered or a wet noodle.

**Comment by eriktq1** [comment]
No, i get it, its wonderful work as it is, im just happy to have it. Just
thought of it, its a little bit funny tho to see Amarr just chew thru TRI
like its nothing when i think of it.
Just noticed that after 7.00 the textures for the eve ships seems to have
disappeared, got any tips on how to solve it? Tried removing and adding the
mods but it did nothing.

Thanks for all your good work, again.

**Comment by DeadAirRT** [comment]
I haven't uploaded the 7.0 versions yet, egosoft renamed a few of the textures used for dirt/rust etc.
